### PR TITLE
Added UG document for AI Assist View Header

### PR DESCRIPTION
**Pull request by RavikumarAkuthotaSf4250**
Added UG document for AI Assist View Header

### CODE REVIEW

File: MAUI/AIAssistView/header.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+platform: MAUI
+control: SfAIAssistView
+documentation: ug
+---
+
+# Header in .NET MAUI AI Assist View (SfAIAssistView)
+
+This section explains how to define and customize the header in the `SfAIAssistView`.
+
+## Show Header
```
Comment (PiruthivirajM) [inline MAUI/AIAssistView/header.md:14]: Show/hide header

File: MAUI/AIAssistView/header.md [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+        this.Content = sfAIAssistView; 
+     } 
+
+{% endhighlight %}
+{% endtabs %}
+
+
+## Header text
+
+The `SfAIAssistView` control allows you to customize the header text using the `HeaderText` property. The default value of the HeaderText is `How can I help you?`.
```
Comment (PiruthivirajM) [inline MAUI/AIAssistView/header.md:40]: remove default string.

File: MAUI/AIAssistView/header.md [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+
+{% endhighlight %}
+{% endtabs %}
+
+## Header customization
+
+The `SfAIAssistView` control allows you to fully customize the header's appearance by using the `HeaderTemplate` property. This property lets you define a custom layout and style for the header.
+
+{% tabs %}
+{% highlight xaml tabtitle="MainPage.xaml" hl_lines="46" %}
```
Comment (PiruthivirajM) [inline MAUI/AIAssistView/header.md:70]: seems highlight line index is not correct.

**Review by PiruthivirajM — COMMENTED** [review]
Resolve comments.

File: MAUI/AIAssistView/header.md (hunk 1 above)
Comment (Lakshminatarajan) [reply MAUI/AIAssistView/header.md]: Addressed review comments.

File: MAUI/AIAssistView/header.md (hunk 3 above)
Comment (Lakshminatarajan) [reply MAUI/AIAssistView/header.md]: Addressed review comments.

File: MAUI/AIAssistView/header.md (hunk 2 above)
Comment (Lakshminatarajan) [reply MAUI/AIAssistView/header.md]: Addressed review comments.

**Review by PiruthivirajM — APPROVED** [review]
Fine

**Pull request merged**